src/shared/pe-binary.c

```diff
@@ -58,7 +58,7 @@ const IMAGE_SECTION_HEADER* pe_section_table_find(
 
         FOREACH_ARRAY(section, sections, n_sections)
                 if (memcmp(section->Name, name, n) == 0 &&
-                    memeqzero(section->Name + n, sizeof(section->Name) - n))
+                    (n == sizeof(sections[0].Name) || memeqzero(section->Name + n, sizeof(section->Name) - n)))
```

Comment:
seems like something to address in memeqzero() to make this clean?

Reply:
i.e. exit early in memeqzero when the size is zero?

Reply:
Here, problem is not in `memeqzero()`. The result of `section->Name + sizeof(section[0].Name)` is undefined, strictly speaking. Hence, we cannot pass it to `memeqzero()`.